flex-checks/src/main/java/org/sonar/flex/checks/FunctionComplexityCheck.java

```diff
@@ -47,7 +47,9 @@
 
   @Override
   public void visitNode(AstNode node) {
-    int functionComplexity = ComplexityVisitor.functionComplexity(node);
+    // By definition      : complexity = number of conditional branches + 1
+    // or, in other words : complexity = sum('if', 'for', '&&', '||', ternary operator, 'while', 'for', 'case') + 1
+    int functionComplexity = ComplexityVisitor.functionComplexity(node) + 1;
```

Comment:
@amelie-renard-sonarsource no need for the extra `+ 1`, it already happens in the visitor.
```suggestion
    int functionComplexity = ComplexityVisitor.functionComplexity(node);
```